template-tags.php

```diff
@@ -417,22 +417,61 @@ function coauthors_ids( $between = null, $betweenLast = null, $before = null, $a
 	), null, $echo );
 }
 
-function get_the_coauthor_meta( $field ) {
-	global $wp_query, $post;
+/**
+ * Outputs the co-authors Meta Data
+ *
+ * @param string $field Required The user field to retrieve.[login, email, nicename, display_name, url, type]
+ * @param string $user_id Optional The user ID for meta
+ *
+ * @return array $meta Value of the user field
+ */
+function get_the_coauthor_meta( $field, $user_id = false ) {
+	global $coauthors_plus;
+
+	if ( ! $user_id ) {
+		$coauthors = get_coauthors();
+    } else {
```

Comment:
Spacing is a little off here - would be good to clean that up.

Reply:
@philipjohn : I have made space alignment changes here. Could you please check and let me know if everything is ok?